CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.6.0
+
+Breaking bug fix:
```

Comment:
Nitpick: not sure I'd categorise this as a bug fix, feels more like an interface change to me!